microcosm_pubsub/daemon.py

```diff
@@ -12,19 +14,25 @@
 
 class ConsumerDaemon(Daemon):
 
-    def __init__(self, event=None):
+    def __init__(self, enable_lambda_mode=False):
         super().__init__()
         self.bound_handlers = None
-        self.sqs_event = event or {}
+        self.enable_lambda_mode = enable_lambda_mode
 
     def make_arg_parser(self):
         parser = super().make_arg_parser()
+
         parser.add_argument("--stdin", action="store_true")
         parser.add_argument("--sqs-queue-url")
         parser.add_argument("--envelope", choices=[
             envelope_cls.__name__
             for envelope_cls in SQSEnvelope.__subclasses__()
         ])
+
+        if self.enable_lambda_mode:
+            # during lambda execution, we do not need to parse arguments, so adding a below catch all args list
+            parser.add_argument('args', nargs=argparse.REMAINDER)
```

Comment:
during lambda execution, we do not need to parse arguments, so adding a below catch all args list